restli-common/src/main/java/com/linkedin/restli/internal/common/URIParamUtils.java

```diff
@@ -469,6 +470,7 @@ public static void addSortedParams(UriBuilder uriBuilder, DataMap params, Protoc
 
   public static URI replaceQueryParam(URI uri, String queryParam, DataComplex values, DataMap parameters, ProtocolVersion version)
   {
+    // This is only used by BatchFinderResponseBuilder right now
```

Comment:
This is a public method, so you can't necessarily make this assumption.

Reply:
The change is moved to the BatchFinderResponseBuilder class now, and it is only limited in that class use case